litellm/main.py

```diff
@@ -2623,6 +2623,9 @@ def completion(  # type: ignore # noqa: PLR0915
             ## RESPONSE OBJECT
             response = model_response
         elif custom_llm_provider == "ollama":
+            if extra_headers is not None:
```

Comment:
i believe the `headers` field should already have this. 

we just need `extra_headers` passed to an llm provider integration for openai/azure as they use the openai sdk which requires it in that way 

the rest, can just use the headers field - which already updates if 'extra_headers' is passed as a param to the sdk

Reply:
Yes, I was just following the pattern for other providers where `extra_headers` was included in `optional_params`. Are you proposing `optional_params['headers'] = headers` instead of `optional_params['extra_headers'] = extra_headers`?